### PR TITLE
[validator] Correct APIs

### PR DESCRIPTION
**Pull request by TonyPythoneer**
# Improvement to existing type definition

Refer to [validator.js](https://github.com/chriso/validator.js), I added more complete types for `isAlpha` and `isAlphanumeric`.

### CODE REVIEW

File: validator/validator.d.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare namespace ValidatorJS {
+  type Alpha = "ar" | "ar-AE" | "ar-BH" | "ar-DZ" | "ar-EG" | "ar-IQ" | "ar-JO" | "ar-KW" | "ar-LB" | "ar-LY" | "ar-MA" | "ar-QA" | "ar-QM" | "ar-SA" | "ar-SD" | "ar-SY" | "ar-TN" | "ar-YE" | "cs-CZ" | "de-DE" | "en-AU" | "en-GB" | "en-HK" | "en-IN" | "en-NZ" | "en-US" | "en-ZA" | "en-ZM" | "es-ES" | "fr-FR" | "hu-HU" | "nl-NL" | "pl-PL" | "pt-BR" | "pt-PT" | "ru-RU" | "sr-RS" | "sr-RS@latin" | "tr-TR";
```
Comment (IOAyman) [inline validator/validator.d.ts]: How about changing the type name to something like `AlphaLocal`, please?

**Review by IOAyman — CHANGES_REQUESTED** [review]

File: validator/validator.d.ts (hunk 1 above)
Comment (TonyPythoneer) [inline validator/validator.d.ts]: Do you think `AlphaLocale` is better?

Comment (IOAyman) [inline validator/validator.d.ts]: :+1: :)
Because that's what it is, right?

**Review by IOAyman — APPROVED** [review]
@TonyPythoneer thank you for your contribution :smile: 

I have approved these changes. It will be merged soon!

**Comment by TonyPythoneer** [comment]
@IOAyman Thanks for your time and review. :smile:

**Pull request merged**